src/Microsoft.DotNet.Build.Tasks/OpenSourceSign.cs

```diff
@@ -29,6 +29,17 @@ public string AssemblyPath
             set;
         }
 
+        /// <summary>
+        /// Whether the task should fail in the case the file is already signed. If not, it will re-sign.
+        /// By default it does not fail, because this condition can occur simply because the build was 
+        /// interrupted at an inopportune moment.
+        /// </summary>
+        public bool FailIfAlreadySigned
```

Comment:
This might be a dumb question, but what is the default for non-required bool MSBuild Task properties? Is it the same as any regular property?

Reply:
Yes...false